src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/DecryptTestsUsingExplicitPrivateKey.cs

```diff
@@ -36,5 +37,27 @@ public static void DecryptUsingWrongPrivateKeyType()
                 Assert.Throws<CryptographicException>(() => ecms.Decrypt(ecms.RecipientInfos[0], ecdsa));
             }
         }
+
+        [Fact]
+        [OuterLoop(/* Leaks key on disk if interrupted */)]
+        public void TestDecryptSimpleAes256_RsaOaepMd5_Throws()
```

Comment:
This test only passes for explicit private keys. On Windows, when certificate keys are used, that calls in to  `CryptMsgControl` which does support OAEP-MD5. In that particular case, MD5 will "just work" while anything that uses the managed PAL will not.

Reply:
Here's fine, or a "if (is windows && !_explicitKeyOrWhateverItsCalled) { return; }" in the other file.